homeassistant/helpers/entity.py

```diff
@@ -319,7 +319,9 @@ def extra_state_attributes(self) -> Mapping[str, Any] | None:
         Implemented by platform classes. Convention for attribute names
         is lowercase snake_case.
         """
-        return self._attr_extra_state_attributes
+        if hasattr(self, "_attr_extra_state_attributes"):
+            return self._attr_extra_state_attributes
+        return None
```

Comment:
```suggestion
        return getattr(self, "_attr_extra_state_attributes", None)
```
`getattr` with `None` as default should work here

Reply:
Right! Will pick that up when I wake up in the morning :)

Reply:
While sorter in code, the typing becomes lost, as getattr returns Any, so that means I would be casting again. IMHO the current solution is clear for that matter.